.github/workflows/ci.yml

```diff
@@ -1,26 +1,26 @@
 name: continuous-integration
-on:
-  push:
-  pull_request:
-    branches:
-      - master
+on: [push, pull_request]
 
 jobs:
   ci:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.6', '2.7' ]
```

Comment:
We could add `2.5` if the [Gemfile](https://github.com/bernardoamc/minitag/blob/61f4b0f43b132a0b618d1c1933b565035bbc8e98/Gemfile#L7) supports it. The Ruby 2.5 EOL is `2021-03-31`.

Reply:
Good point, I don't think there's anything in the codebase preventing us from running Ruby 2.5 right now, we would need to change the Gemfile. :)